crates/bevy_asset/src/server/mod.rs

```diff
@@ -1286,22 +1286,48 @@ pub enum AssetLoadError {
     },
 }
 
-/// An error that occurs when an [`AssetLoader`] is not registered for a given extension.
 #[derive(Error, Debug, Clone)]
+#[error("Failed to load asset '{path}' with asset loader '{loader_name}': {error}")]
+pub struct AssetLoaderError {
+    path: AssetPath<'static>,
+    loader_name: &'static str,
+    error: Arc<dyn std::error::Error + Send + Sync + 'static>,
+}
+
+impl PartialEq for AssetLoaderError {
+    /// Equality comparison for `AssetLoaderError::error` is partial (only through `TypeId`)
```

Comment:
Probably want to use different phrasing than "partial" here (and in the other location:

>   is partial (only through TypeId)

The "partial" in PartialEq refers to non-reflexiveness, not "partial comparison"

Reply:
replaced "partial" to "not full"